src/createServer.js

```diff
@@ -1,3 +1,41 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { validateURL } = require('./validateUrl');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const formatedPath = req.url.slice(1);
+    const [text, queryString] = formatedPath.split('?');
+
+    const params = new URLSearchParams(queryString);
+    const toCase = params.get('toCase');
+
+    const errors = validateURL(text, toCase);
+
+    if (errors.length) {
+      res.statusCode = 400;
```

Comment:
`If something is not correct, you should respond with 400 status, Bad request statusText`

Check the task requirements